x/ccv/provider/migrations/migrator.go

```diff
@@ -29,6 +30,12 @@ func (m Migrator) Migrate1to2(ctx sdktypes.Context) error {
 
 // Migrate2to3 migrates x/ccvprovider state from consensus version 2 to 3.
 func (m Migrator) Migrate2to3(ctx sdktypes.Context) error {
-	v3.MigrateParams(ctx, m.paramSpace)
```

Comment:
Why is this removed?

Reply:
Because it was moved to v4 (migration used for ICS @ v4.1).

`Migrate2to3` is for v4.0.0.
`Migrate3to4` is for v4.1.0.